site/gatsby-site/playwright/e2e-full/cite.spec.ts

```diff
@@ -567,8 +559,6 @@ test.describe('Cite pages', () => {
 
     test('Should link similar incidents', async ({ page, login }) => {
 
-        test.slow();
```

Comment:
I've found that `tests.slow()` makes things worse by delaying the time to get tests results, and overall it is not necessary, we might want to delete them all later.